### ISSUE TITLE
"Python has stopped working" when starting ab

### ISSUE THREAD

**Issue by xiaoshir** [LCA-ActivityBrowser/activity-browser#377]
I have been using the development version of activity browser (installed in its own environment"ab_dev") for quite a while. But recently I suddenly have this error message when I try to start activity browser, that says
![image](https://user-images.githubusercontent.com/28433916/75437592-bc557c80-5956-11ea-8afa-4d83b904c0c2.png)

I then remove the whole environment, made sure I have the right channels as well as the right order of their sequence and reinstall it again, but I still have the same issue. 

I ran into this issue before (https://github.com/LCA-ActivityBrowser/activity-browser/issues/234#issuecomment-488224722), and "magically" solved it by updating my conda. Unfortunately this does not help this time. Below is a list of packages I have in the environment. What can I do to resolve this?

```
# Name                    Version                   Build  Channel
activity-browser-dev      2020.02.25                 py_0    bsteubing
appdirs                   1.4.3                      py_1    conda-forge
arrow                     0.15.5                   py38_0    conda-forge
asteval                   0.9.16             pyh5ca1d4c_0    conda-forge
astunparse                1.6.2                      py_0    conda-forge
beautifulsoup4            4.8.2                    py38_0    conda-forge
brightway2                2.3                        py_2    cmutel
bw2analyzer               0.9.4                      py_1    cmutel
bw2calc                   1.8.0                      py_0    cmutel
bw2data                   3.6.2                      py_0    cmutel
bw2io                     0.7.12                     py_0    cmutel
bw2parameters             0.6.6                      py_0    cmutel
bw_migrations             0.1                        py_0    cmutel
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py38_0    conda-forge
cffi                      1.13.2           py38hb32ad35_0    conda-forge
chardet                   3.0.4                 py38_1003    conda-forge
click                     7.0                        py_0    conda-forge
cryptography              2.8              py38hb32ad35_1    conda-forge
cycler                    0.10.0                     py_2    conda-forge
decorator                 4.4.1                      py_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
eidl                      1.2.0                      py_0    haasad
eidl7zip                  1.0.0                         1    haasad
eight                     0.4.2                      py_2    conda-forge
fasteners                 0.14.1                     py_3    conda-forge
flask                     1.1.1                      py_1    conda-forge
freetype                  2.10.0               h563cfd7_1    conda-forge
future                    0.18.2                   py38_0    conda-forge
fuzzywuzzy                0.17.0                     py_0    conda-forge
icc_rt                    2019.0.0             h0cc432a_1
icu                       64.2                 he025d50_1    conda-forge
idna                      2.9                        py_1    conda-forge
intel-openmp              2019.4                      245
itsdangerous              1.1.0                      py_0    conda-forge
jinja2                    2.11.1                     py_0    conda-forge
jpeg                      9c                hfa6e2cd_1001    conda-forge
kiwisolver                1.1.0            py38he980bc4_0    conda-forge
libblas                   3.8.0                    14_mkl    conda-forge
libcblas                  3.8.0                    14_mkl    conda-forge
libclang                  9.0.1           default_hf44288c_0    conda-forge
libiconv                  1.15              hfa6e2cd_1005    conda-forge
liblapack                 3.8.0                    14_mkl    conda-forge
libpng                    1.6.37               h7602738_0    conda-forge
libxml2                   2.9.10               h9ce36c8_0    conda-forge
libxslt                   1.1.33               heafd4d3_0    conda-forge
lxml                      4.5.0            py38heafd4d3_0    conda-forge
markupsafe                1.1.1            py38hfa6e2cd_0    conda-forge
matplotlib-base           3.1.3            py38h2981e6d_0    conda-forge
mkl                       2019.4                      245
mkl-service               2.3.0            py38hfa6e2cd_0    conda-forge
monotonic                 1.5                        py_0    conda-forge
mrio_common_metadata      0.1.1                      py_0    cmutel
networkx                  2.4                        py_0    conda-forge
numpy                     1.18.1           py38hc71023c_0    conda-forge
openssl                   1.1.1d               hfa6e2cd_0    conda-forge
pandas                    1.0.1            py38he350917_0    conda-forge
patsy                     0.5.1                      py_0    conda-forge
peewee                    3.10.0           py38ha85dd04_0    conda-forge
pip                       20.0.2                     py_2    conda-forge
presamples                0.2.5                h39e3cac_0    pascallesage
psutil                    5.7.0            py38hfa6e2cd_0    conda-forge
pycparser                 2.19                       py_2    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pypardiso                 0.2.2                      py_0    haasad
pyparsing                 2.4.6                      py_0    conda-forge
pyprind                   2.11.2                py38_1000    conda-forge
pyside2                   5.13.2           py38hfa7ce6d_2    conda-forge
pysocks                   1.7.1                    py38_0    conda-forge
python                    3.8.1                he1f5543_2    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-levenshtein        0.12.0          py38hfa6e2cd_1001    conda-forge
pytz                      2019.3                     py_0    conda-forge
qt                        5.12.5               h7ef1ec2_0    conda-forge
requests                  2.23.0                   py38_0    conda-forge
scipy                     1.3.2            py38h582fac2_0    conda-forge
seaborn                   0.10.0                     py_1    conda-forge
setuptools                45.2.0                   py38_0    conda-forge
six                       1.14.0                   py38_0    conda-forge
soupsieve                 1.9.4                    py38_0    conda-forge
sqlite                    3.30.1               hfa6e2cd_0    conda-forge
stats_arrays              0.6.4                      py_0    cmutel
statsmodels               0.11.1           py38hfa6e2cd_0    conda-forge
tornado                   6.0.3            py38hfa6e2cd_4    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
unidecode                 1.1.1                      py_0    conda-forge
urllib3                   1.25.7                   py38_0    conda-forge
vc                        14.1                 h0510ff6_4
voluptuous                0.11.7                     py_0    conda-forge
vs2015_runtime            14.16.27012          hf0eaf9b_1
werkzeug                  1.0.0                      py_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
whoosh                    2.7.4                      py_1    conda-forge
win_inet_pton             1.1.0                    py38_0    conda-forge
wincertstore              0.2                   py38_1003    conda-forge
wrapt                     1.12.0           py38hfa6e2cd_0    conda-forge
xlrd                      1.2.0                      py_0    conda-forge
xlsxwriter                1.2.8                      py_0    conda-forge
zlib                      1.2.11            h2fa13f4_1006    conda-forge
```

**Comment by dgdekoning** [comment]
Hey, thanks for the comprehensive bug report!

I was able to completely recreate your environment but I was not able to recreate the exception :(.
I have noticed it myself several times while I was developing the code but it usually went away on its own. The greater internet attributes the 'python has stopped working' exception with Qt (PyQt in the case of anaconda navigator / spyder).

Can I ask you to create an environment using the 'defaults' channel? Perhaps that offers improvement.
```bash
conda create -n abdefault -c defaults -c conda-forge -c cmutel -c bsteubing -c haasad -c pascallesage activity-browser-dev
```
<details><summary>Should result in the following environment</summary>
<p>

```terminal
# Name                    Version                   Build  Channel
activity-browser-dev      2020.02.25                 py_0    bsteubing
appdirs                   1.4.3            py37h28b3542_0
arrow                     0.13.1                   py37_0
asn1crypto                1.3.0                    py37_0
asteval                   0.9.16             pyh5ca1d4c_0    conda-forge
astunparse                1.6.2                      py_0    conda-forge
beautifulsoup4            4.8.2                    py37_0
blas                      1.0                         mkl
brightway2                2.3                        py_2    cmutel
bw2analyzer               0.9.4                      py_1    cmutel
bw2calc                   1.8.0                      py_0    cmutel
bw2data                   3.6.2                      py_0    cmutel
bw2io                     0.7.12                     py_0    cmutel
bw2parameters             0.6.6                      py_0    cmutel
bw_migrations             0.1                        py_0    cmutel
ca-certificates           2020.1.1                      0
certifi                   2019.11.28               py37_0
cffi                      1.14.0           py37h7a1dbc1_0
chardet                   3.0.4                 py37_1003
click                     7.0                      py37_0
cryptography              2.8              py37h7a1dbc1_0
cycler                    0.10.0                   py37_0
decorator                 4.4.1                      py_0
docopt                    0.6.2                    py37_0
eidl                      1.2.0                      py_0    haasad
eidl7zip                  1.0.0                         1    haasad
eight                     0.4.2                      py_2    conda-forge
fasteners                 0.15                       py_0
flask                     1.1.1                      py_0
freetype                  2.9.1                ha9979f8_1
future                    0.18.2                   py37_0
fuzzywuzzy                0.17.0                     py_0    conda-forge
icc_rt                    2019.0.0             h0cc432a_1
icu                       64.2                 he025d50_1    conda-forge
idna                      2.8                      py37_0
intel-openmp              2020.0                      166
itsdangerous              1.1.0                    py37_0
jinja2                    2.11.1                     py_0
jpeg                      9c                hfa6e2cd_1001    conda-forge
kiwisolver                1.1.0            py37ha925a31_0
libclang                  9.0.1           default_hf44288c_0    conda-forge
libiconv                  1.15                 h1df5818_7
libpng                    1.6.37               h2a8f88b_0
libxml2                   2.9.10               h9ce36c8_0    conda-forge
libxslt                   1.1.33               h579f668_0
lxml                      4.5.0            py37h1350720_0
markupsafe                1.1.1            py37he774522_0
matplotlib                3.1.3                    py37_0
matplotlib-base           3.1.3            py37h64f37c6_0
mkl                       2020.0                      166
mkl-service               2.3.0            py37hb782905_0
mkl_fft                   1.0.15           py37h14836fe_0
mkl_random                1.1.0            py37h675688f_0
monotonic                 1.5                        py_0
mrio_common_metadata      0.1.1                      py_0    cmutel
networkx                  2.4                        py_0
numpy                     1.18.1           py37h93ca92e_0
numpy-base                1.18.1           py37hc3f5095_1
openssl                   1.1.1d               he774522_4
pandas                    1.0.1            py37h47e9c7a_0
peewee                    3.10.0           py37ha85dd04_0    conda-forge
pip                       20.0.2                   py37_1
presamples                0.2.7            py37h39e3cac_0    pascallesage
psutil                    5.6.7            py37he774522_0
pycparser                 2.19                     py37_0
pyopenssl                 19.1.0                   py37_0
pypardiso                 0.2.2                      py_0    haasad
pyparsing                 2.4.6                      py_0
pyprind                   2.11.2                py37_1000    conda-forge
pyqt                      5.12.3           py37h6538335_1    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pyside2                   5.13.2           py37hfa7ce6d_2    conda-forge
pysocks                   1.7.1                    py37_0
python                    3.7.6                h60c2a47_2
python-dateutil           2.8.1                      py_0
python-levenshtein        0.12.0          py37he774522_1001
pytz                      2019.3                     py_0
qt                        5.12.5               h7ef1ec2_0    conda-forge
requests                  2.22.0                   py37_1
scipy                     1.4.1            py37h9439919_0
seaborn                   0.10.0                     py_0
setuptools                45.2.0                   py37_0
six                       1.14.0                   py37_0
soupsieve                 1.9.5                    py37_0
sqlite                    3.31.1               he774522_0
stats_arrays              0.6.4                      py_0    cmutel
tornado                   6.0.3            py37he774522_3
unicodecsv                0.14.1                   py37_0
unidecode                 1.1.1                      py_0
urllib3                   1.25.8                   py37_0
vc                        14.1                 h0510ff6_4
voluptuous                0.11.7                     py_0    conda-forge
vs2015_runtime            14.16.27012          hf0eaf9b_1
werkzeug                  1.0.0                      py_0
wheel                     0.34.2                   py37_0
whoosh                    2.7.4                    py37_1
win_inet_pton             1.1.0                    py37_0
wincertstore              0.2                      py37_0
wrapt                     1.11.2           py37he774522_0
xlrd                      1.2.0                    py37_0
xlsxwriter                1.2.7                      py_0
zlib                      1.2.11               h62dcd97_3
```

</p>
</details>

**Comment by xiaoshir** [comment]
Thank you for the help. I followed the advice, but the issue remain. I noticed the packages installed are exactly the same as the one you listed above. What other reason that could cause this? So strange...

```
(abdefault) C:\Users\zhang_x>conda list
# packages in environment at C:\miniconda3_py37\envs\abdefault:
#
# Name                    Version                   Build  Channel
activity-browser-dev      2020.02.25                 py_0    bsteubing
appdirs                   1.4.3            py37h28b3542_0
arrow                     0.13.1                   py37_0
asn1crypto                1.3.0                    py37_0
asteval                   0.9.16             pyh5ca1d4c_0    conda-forge
astunparse                1.6.2                      py_0    conda-forge
beautifulsoup4            4.8.2                    py37_0
blas                      1.0                         mkl
brightway2                2.3                        py_2    cmutel
bw2analyzer               0.9.4                      py_1    cmutel
bw2calc                   1.8.0                      py_0    cmutel
bw2data                   3.6.2                      py_0    cmutel
bw2io                     0.7.12                     py_0    cmutel
bw2parameters             0.6.6                      py_0    cmutel
bw_migrations             0.1                        py_0    cmutel
ca-certificates           2020.1.1                      0
certifi                   2019.11.28               py37_0
cffi                      1.14.0           py37h7a1dbc1_0
chardet                   3.0.4                 py37_1003
click                     7.0                      py37_0
cryptography              2.8              py37h7a1dbc1_0
cycler                    0.10.0                   py37_0
decorator                 4.4.1                      py_0
docopt                    0.6.2                    py37_0
eidl                      1.2.0                      py_0    haasad
eidl7zip                  1.0.0                         1    haasad
eight                     0.4.2                      py_2    conda-forge
fasteners                 0.15                       py_0
flask                     1.1.1                      py_0
freetype                  2.9.1                ha9979f8_1
future                    0.18.2                   py37_0
fuzzywuzzy                0.17.0                     py_0    conda-forge
icc_rt                    2019.0.0             h0cc432a_1
icu                       64.2                 he025d50_1    conda-forge
idna                      2.8                      py37_0
intel-openmp              2020.0                      166
itsdangerous              1.1.0                    py37_0
jinja2                    2.11.1                     py_0
jpeg                      9c                hfa6e2cd_1001    conda-forge
kiwisolver                1.1.0            py37ha925a31_0
libclang                  9.0.1           default_hf44288c_0    conda-forge
libiconv                  1.15                 h1df5818_7
libpng                    1.6.37               h2a8f88b_0
libxml2                   2.9.10               h9ce36c8_0    conda-forge
libxslt                   1.1.33               h579f668_0
lxml                      4.5.0            py37h1350720_0
markupsafe                1.1.1            py37he774522_0
matplotlib                3.1.3                    py37_0
matplotlib-base           3.1.3            py37h64f37c6_0
mkl                       2020.0                      166
mkl-service               2.3.0            py37hb782905_0
mkl_fft                   1.0.15           py37h14836fe_0
mkl_random                1.1.0            py37h675688f_0
monotonic                 1.5                        py_0
mrio_common_metadata      0.1.1                      py_0    cmutel
networkx                  2.4                        py_0
numpy                     1.18.1           py37h93ca92e_0
numpy-base                1.18.1           py37hc3f5095_1
openssl                   1.1.1d               he774522_4
pandas                    1.0.1            py37h47e9c7a_0
peewee                    3.10.0           py37ha85dd04_0    conda-forge
pip                       20.0.2                   py37_1
presamples                0.2.7            py37h39e3cac_0    pascallesage
psutil                    5.6.7            py37he774522_0
pycparser                 2.19                     py37_0
pyopenssl                 19.1.0                   py37_0
pypardiso                 0.2.2                      py_0    haasad
pyparsing                 2.4.6                      py_0
pyprind                   2.11.2                py37_1000    conda-forge
pyqt                      5.12.3           py37h6538335_1    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pyside2                   5.13.2           py37hfa7ce6d_2    conda-forge
pysocks                   1.7.1                    py37_0
python                    3.7.6                h60c2a47_2
python-dateutil           2.8.1                      py_0
python-levenshtein        0.12.0          py37he774522_1001
pytz                      2019.3                     py_0
qt                        5.12.5               h7ef1ec2_0    conda-forge
requests                  2.22.0                   py37_1
scipy                     1.4.1            py37h9439919_0
seaborn                   0.10.0                     py_0
setuptools                45.2.0                   py37_0
six                       1.14.0                   py37_0
soupsieve                 1.9.5                    py37_0
sqlite                    3.31.1               he774522_0
stats_arrays              0.6.4                      py_0    cmutel
tornado                   6.0.3            py37he774522_3
unicodecsv                0.14.1                   py37_0
unidecode                 1.1.1                      py_0
urllib3                   1.25.8                   py37_0
vc                        14.1                 h0510ff6_4
voluptuous                0.11.7                     py_0    conda-forge
vs2015_runtime            14.16.27012          hf0eaf9b_1
werkzeug                  1.0.0                      py_0
wheel                     0.34.2                   py37_0
whoosh                    2.7.4                    py37_1
win_inet_pton             1.1.0                    py37_0
wincertstore              0.2                      py37_0
wrapt                     1.11.2           py37he774522_0
xlrd                      1.2.0                    py37_0
xlsxwriter                1.2.7                      py_0
zlib                      1.2.11               h62dcd97_3
```

**Comment by dgdekoning** [comment]
Will the crash also occur if you run python by itself?
If so, do you have other conda environments where python does work?

**Comment by xiaoshir** [comment]
No it won't crash if I just run python in this env. I also have other conda environments where python, brightway, jupyter notebook... work fine. I usually have two environments running at the same time, one with activity browser, the other with brightway+jupyter, etc.

**Comment by xiaoshir** [comment]
Since I want to make it to work, I tried to follow the advice under [https://github.com/LCA-ActivityBrowser/activity-browser#important-notice-the-ab-switched-from-pyqt5-to-pyside2](url)

I tried the 2nd option: 2. Remove PyQt5 and install PySide2 (within your existing environment)

and saw this
```
(ab_dev) C:\Users\zhang_x>conda install -c conda-forge pyside2 matplotlib-base seaborn
Collecting package metadata (current_repodata.json): done
Solving environment: |
The environment is inconsistent, please check the package plan carefully
The following packages are causing the inconsistency:

  - bsteubing/noarch::activity-browser-dev==2019.10.30=py_0
done
```

**Comment by dgdekoning** [comment]
Can you look at the package list again? I have a feeling the `install` went and removed a lot of other packages on the side.

`bsteubing/noarch::activity-browser-dev==2019.10.30=py_0` has been a pain ever since the switch to pyside2.

Just to be sure that Qt is the problem, can I ask you to create two environments, one with PyQt5 and one with PySide2.
`conda create -n pyqt pyqt==5.9.2`
`conda create -n pyside pyside2==5.13.2`

And then try and import a qt module in python `from PyQt5 import QtWidgets` and `from PySide2 import QtWidgets`
Either one or both should cause python to crash.

**Comment by xiaoshir** [comment]
List of packages in ab_dev:
```
(ab_dev) C:\Users\zhang_x>conda list
# packages in environment at C:\miniconda3_py37\envs\ab_dev:
#
# Name                    Version                   Build  Channel
activity-browser-dev      2020.02.25                 py_0    bsteubing
appdirs                   1.4.3                      py_1    conda-forge
arrow                     0.15.5                   py38_0    conda-forge
asteval                   0.9.16             pyh5ca1d4c_0    conda-forge
astunparse                1.6.2                      py_0    conda-forge
beautifulsoup4            4.8.2                    py38_0    conda-forge
brightway2                2.3                        py_2    cmutel
bw2analyzer               0.9.4                      py_1    cmutel
bw2calc                   1.8.0                      py_0    cmutel
bw2data                   3.6.2                      py_0    cmutel
bw2io                     0.7.12                     py_0    cmutel
bw2parameters             0.6.6                      py_0    cmutel
bw_migrations             0.1                        py_0    cmutel
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py38_0    conda-forge
cffi                      1.13.2           py38hb32ad35_0    conda-forge
chardet                   3.0.4                 py38_1003    conda-forge
click                     7.0                        py_0    conda-forge
cryptography              2.8              py38hb32ad35_1    conda-forge
cycler                    0.10.0                     py_2    conda-forge
decorator                 4.4.1                      py_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
eidl                      1.2.0                      py_0    haasad
eidl7zip                  1.0.0                         1    haasad
eight                     0.4.2                      py_2    conda-forge
fasteners                 0.14.1                     py_3    conda-forge
flask                     1.1.1                      py_1    conda-forge
freetype                  2.10.0               h563cfd7_1    conda-forge
future                    0.18.2                   py38_0    conda-forge
fuzzywuzzy                0.17.0                     py_0    conda-forge
icc_rt                    2019.0.0             h0cc432a_1
icu                       64.2                 he025d50_1    conda-forge
idna                      2.9                        py_1    conda-forge
intel-openmp              2019.4                      245
itsdangerous              1.1.0                      py_0    conda-forge
jinja2                    2.11.1                     py_0    conda-forge
jpeg                      9c                hfa6e2cd_1001    conda-forge
kiwisolver                1.1.0            py38he980bc4_0    conda-forge
libblas                   3.8.0                    14_mkl    conda-forge
libcblas                  3.8.0                    14_mkl    conda-forge
libclang                  9.0.1           default_hf44288c_0    conda-forge
libiconv                  1.15              hfa6e2cd_1005    conda-forge
liblapack                 3.8.0                    14_mkl    conda-forge
libpng                    1.6.37               h7602738_0    conda-forge
libxml2                   2.9.10               h9ce36c8_0    conda-forge
libxslt                   1.1.33               heafd4d3_0    conda-forge
lxml                      4.5.0            py38heafd4d3_0    conda-forge
markupsafe                1.1.1            py38hfa6e2cd_0    conda-forge
matplotlib-base           3.1.3            py38h2981e6d_0    conda-forge
mkl                       2019.4                      245
mkl-service               2.3.0            py38hfa6e2cd_0    conda-forge
monotonic                 1.5                        py_0    conda-forge
mrio_common_metadata      0.1.1                      py_0    cmutel
networkx                  2.4                        py_0    conda-forge
numpy                     1.18.1           py38hc71023c_0    conda-forge
openssl                   1.1.1d               hfa6e2cd_0    conda-forge
pandas                    1.0.1            py38he350917_0    conda-forge
patsy                     0.5.1                      py_0    conda-forge
peewee                    3.10.0           py38ha85dd04_0    conda-forge
pip                       20.0.2                     py_2    conda-forge
presamples                0.2.5                h39e3cac_0    pascallesage
psutil                    5.7.0            py38hfa6e2cd_0    conda-forge
pycparser                 2.19                       py_2    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pypardiso                 0.2.2                      py_0    haasad
pyparsing                 2.4.6                      py_0    conda-forge
pyprind                   2.11.2                py38_1000    conda-forge
pyside2                   5.13.2           py38hfa7ce6d_2    conda-forge
pysocks                   1.7.1                    py38_0    conda-forge
python                    3.8.1                he1f5543_2    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-levenshtein        0.12.0          py38hfa6e2cd_1001    conda-forge
pytz                      2019.3                     py_0    conda-forge
qt                        5.12.5               h7ef1ec2_0    conda-forge
requests                  2.23.0                   py38_0    conda-forge
scipy                     1.3.2            py38h582fac2_0    conda-forge
seaborn                   0.10.0                     py_1    conda-forge
setuptools                45.2.0                   py38_0    conda-forge
six                       1.14.0                   py38_0    conda-forge
soupsieve                 1.9.4                    py38_0    conda-forge
sqlite                    3.30.1               hfa6e2cd_0    conda-forge
stats_arrays              0.6.4                      py_0    cmutel
statsmodels               0.11.1           py38hfa6e2cd_0    conda-forge
tornado                   6.0.3            py38hfa6e2cd_4    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
unidecode                 1.1.1                      py_0    conda-forge
urllib3                   1.25.7                   py38_0    conda-forge
vc                        14.1                 h0510ff6_4
voluptuous                0.11.7                     py_0    conda-forge
vs2015_runtime            14.16.27012          hf0eaf9b_1
werkzeug                  1.0.0                      py_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
whoosh                    2.7.4                      py_1    conda-forge
win_inet_pton             1.1.0                    py38_0    conda-forge
wincertstore              0.2                   py38_1003    conda-forge
wrapt                     1.12.0           py38hfa6e2cd_0    conda-forge
xlrd                      1.2.0                      py_0    conda-forge
xlsxwriter                1.2.8                      py_0    conda-forge
zlib                      1.2.11            h2fa13f4_1006    conda-forge
```
Followed your advice and created pyqt with the specific version installed, and I could import the qt module in this env without the problem. So I guess the problem is not qt.

Had problem creating and installing the 2nd env though, due to missing channel
```
(base) C:\Users\zhang_x>conda create -n pyside pyside==5.13.2
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - pyside==5.13.2

Current channels:

  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch
  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/cmutel/win-64
  - https://conda.anaconda.org/cmutel/noarch
  - https://conda.anaconda.org/bsteubing/win-64
  - https://conda.anaconda.org/bsteubing/noarch
  - https://conda.anaconda.org/haasad/win-64
  - https://conda.anaconda.org/haasad/noarch
  - https://conda.anaconda.org/pascallesage/win-64
  - https://conda.anaconda.org/pascallesage/noarch
  - https://conda.anaconda.org/konstantinstadler/win-64
  - https://conda.anaconda.org/konstantinstadler/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Comment by dgdekoning** [comment]
Ahhh, yes. The `pyside==5.13.2` is supposed to be `pyside2==5.13.2`.

**Comment by dgdekoning** [comment]
To be honest, I am mostly guessing at this point. I have absolutely no idea why python crashes on your system.

**Comment by xiaoshir** [comment]
Sorry I missded "2" in "pyside". Had no issue importing the module as well. 

hmmm. thanks for all your suggestions anyway. I know the difficult part here is you can not recreate my exception. Trying to recall what did I do before this issue occurred.

But you see I spent some effort trying to resolve this as ab is a good tool:)

**Comment by dgdekoning** [comment]
Thanks for trying that anyway! I'll keep track of this and will make sure to record any instances of python crashing while I'm working on the code.

**Comment by romainsacchi** [comment]
Hello,
sadly, I am now reporting the exact same issue while starting the development version.
It is very strange as it runs fine on my home computer.
Maybe I should add @xiaoshir and I are in the same organisation and that we likely have similar firewall rules, antivirus and the like, as well as having the same OS (Win 10 Enterprise, 64 bit).

**Comment by romainsacchi** [comment]
In the Windows log, we get this:
Faulting application name: python.exe, version: 3.7.6150.1013, time stamp: 0x5e5fd786
Faulting module name: ig9icd64.dll, version: 25.20.100.6472, time stamp: 0x5c0ebd3e
Exception code: 0xc0000005
Fault offset: 0x00000000000202ee
Faulting process id: 0x239c
Faulting application start time: 0x01d5f2d0b2962776
Faulting application path: C:\Users\sacchi_r\AppData\Local\Continuum\miniconda3\envs\ab\python.exe
Faulting module path: C:\Windows\System32\DriverStore\FileRepository\ki130624.inf_amd64_d85487143eec5e1a\ig9icd64.dll
Report Id: cc8b9ec2-8f48-47ff-8521-0d74b6f99364
Faulting package full name: 
Faulting package-relative application ID:

**Comment by romainsacchi** [comment]
It says here (https://stackoverflow.com/questions/47828821/python-has-stopped-working) that the error code 0xc0000005 is an 'memory access violation' error. Here it probably relates to ig9icd64.dll. So it seems to be a Windows issue, not one related to ab :-/

**Comment by dgdekoning** [comment]
Sad to hear that it affects multiple people in the same location / organization. So if this is a windows issue it might go away in the next update of drivers?

Found a reference to a similar kind of issue (Qt, windows and the ig9cid64.dll) [here](https://github.com/nextcloud/desktop/issues/896).

**Comment by xiaoshir** [comment]
@romainsacchi frankly speaking, I'm even a bit glad to see your face here 😄 ....as I recall I got this error right after we managed installing [rmnd-lca ](https://github.com/romainsacchi/rmnd-lca) on my laptop... so this error must have something to do with that.

@dgdekoning rmnd-lca is a package Romain developed, and I just got it installed last week. After that, I encountered this error when starting activity browser, which I did't have before.
